packages/mui-joy/src/Avatar/Avatar.tsx

```diff
@@ -143,7 +143,7 @@ const Avatar = React.forwardRef(function Avatar(inProps, ref) {
     color = 'neutral',
     component = 'div',
     size = 'md',
-    variant = 'contained',
+    variant = 'light',
```

Comment:
Is there a reason for this?

Reply:
One intention of the global variant is to bring attention. Currently `contained` brings the most attention and `text` be the last. My feeling is that Avatar is a complimentary UI, so it should not be `contained` by default.

Reply:
Gottcha gottcha!